nemo/collections/nlp/models/language_modeling/megatron_lm_encoder_decoder_model.py

```diff
@@ -822,7 +822,11 @@ def build_pretraining_data_loader(self, dataset, consumed_samples, num_workers):
 
         # Torch dataloader.
         return torch.utils.data.DataLoader(
-            dataset, batch_sampler=batch_sampler, num_workers=num_workers, pin_memory=True,
+            dataset,
+            batch_sampler=batch_sampler,
+            num_workers=num_workers,
+            pin_memory=True,
+            persistent_workers=True if self.cfg.data.num_workers > 0 else False,
```

Comment:
Approved. Can you test it passes with text memmap and num_workers > 0?

Reply:
persistent_workers should not have any effect there, so resolving this